templates/dashboard/customer/list.html

```diff
@@ -1,6 +1,7 @@
 {% extends "dashboard/base.html" %}
 {% load i18n %}
 {% load prices_i18n %}
+{% load shop %}
```

Comment:
If you're using only one function, it would be more convenient to use `{% load function from shop %}`

Reply:
Will do. But I think we need PR which will make those changes for every template for every templatetag library